docs/templates/50-configuring.ftl

```diff
@@ -0,0 +1,26 @@
+# Configuring
+
+Once the ${solution_name} plugin is installed, before you can use it in a job, you need to configure:
+
+1. ${polaris_cli_name}
+1. ${solution_name}
+
+Use the following process to configure ${polaris_cli_name}:
+
+1. In Jenkins, navigate to *Manage Jenkins > Global Tool Configuration*.
+1. Scroll down to the ${cli_config_header} section, click ${add_polaris_cli_button}, and complete the following:
+* *Name*: A name for the ${polaris_cli_name} installation.
```

Comment:
I think it would be worthwhile to talk about choosing a tool installer here-- they can select either "Install from Polaris" (default) to install it from their configured Polaris instance or they can select another Jenkins provided/Jenkins Plugin provided installer for a custom solution (like, say, unzipping a local installation or whatever crazy shenanigans they want to do)